adb_graphics/specs.py

```diff
@@ -188,6 +188,26 @@ def lifted_index_colors(self) -> np.ndarray:
         ctable[15] = [1, 1, 1, 1]
         return ctable
 
+    @property
+    @lru_cache()
+    def mdn_colors(self) -> np.ndarray:
+
+        ''' Default color map for Max Downdraft '''
+
+        grays = cm.get_cmap('Greys', 2)([0])
+        others = cm.get_cmap('jet', 18)(range(18, 1, -1), alpha=0.6)
+        return np.concatenate((others, grays))
+    @property
+    @lru_cache()
+
+    def mup_colors(self) -> np.ndarray:
```

Comment:
Please rearrange the whitespace here to match the others. This should also be moved down to be in order with the next method.